arangod/Replication/Syncer.cpp

```diff
@@ -147,11 +147,12 @@ arangodb::Result applyCollectionDumpMarkerInternal(
 
     try {
       OperationResult opRes(Result(), options);
+      auto potentiallyConflictingKey = std::string{};
```

Comment:
Why do we have a `conflictingDocumentKey` variable _and_ a `potentiallyConflictingDocumentKey` variable here? Can't we just use a single variable for this?

Reply:
The though behind it was that I wanted the "return" variable `conflictingDocumentKey` to only be set/changed in case it contains a meaningful value, and added this corresponding assertion in `TailingSyncer.cpp`:
```
TRI_ASSERT(res.is(TRI_ERROR_ARANGO_TRY_AGAIN) == !conflictDocumentKey.empty());
```